backend/cmd/server/main.go

```diff
@@ -147,9 +152,30 @@ func startServer(logger *log.Logger, cfg *config.Config, mux *http.ServeMux, thu
 	server := &http.Server{
 		Handler:           wrappedMux,
 		ReadHeaderTimeout: 10 * time.Second, // Mitigate Slowloris attacks
+		WriteTimeout:      10 * time.Second,
+		IdleTimeout:       120 * time.Second,
 	}
 
 	if err := server.Serve(ln); err != nil {
 		logger.Fatal("Failed to serve requests", log.Error(err))
 	}
 }
+
+// startHTTPServer starts the HTTP server without TLS.
+func startHTTPServer(logger *log.Logger, cfg *config.Config, mux *http.ServeMux) {
```

Comment:
We need to wrap `AccessLogHandler` for this case also. Shall we move that logic to a common place in main?

Reply:
Fixed with https://github.com/asgardeo/thunder/compare/90c829233870c24ac12860112f41c4886d70a018..2188fc128c8aacba528eba6f0d1fc7ab5935ea69 and improved with https://github.com/asgardeo/thunder/compare/2188fc128c8aacba528eba6f0d1fc7ab5935ea69..99d202e98e509f80e82f0aef03d9670c484d57c9